### DISCUSSION TITLE
After the 1.0 release, what should be the main focus?

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#370] [General]
Everyone can offer any suggestions to help us plan the future development roadmap for Ant Design Web3.

**Comment by yutingzhao1991** [comment]
Here are some important matters I've considered, ranked in order of priority:

- Enhance the ConnectModal experience; connecting a wallet is the most critical user experience link. We need to support a variety of wallet connections better and more comprehensively. In addition to the current simple and normal modes, we should provide a better interactive experience.
- Add more icons. Icons are simple yet very practical, and we should support mainstream blockchain and token logos.
- Support a non-EVM chain, such as Bitcoin.
- Complete the tutorials to enrich the project content.
- Support internationalization, such as adding Chinese language support.
- In line with community hotspots and needs, add more practical components.

---


目前我想到的一些重要事项，优先级从高到低排列：

- 提升 ConnectModal 的体验，连接钱包是最关键的用户体验链路。要更好更多的支持各种钱包的连接。除了当前的 simple 和 normal 的模式外提供更好体验的交互。
- 添加更多的 icon，icon 简单但是又很实用，可以把主流的链和代币的图标都支持上。
- 支持一条非 EVM 的链，比如比特币。
- 完成教程，丰富项目内容。
- 支持国际化，比如支持下中文。
- 结合社区热点和需求，添加更多实用的组件。

**Reply by Likang0122** [reply]
感觉在移动端的体验不是很好，手机钱包大多都有内置dapp浏览器，现在的流程是不是我在钱包里打开dapp了，我还要用我另一个装有钱包的手机扫码链接

**Reply by yutingzhao1991** [reply]
这个考虑了的，移动端可以直接点下面这个按钮：

![image](https://github.com/ant-design/ant-design-web3/assets/1061968/5a96d219-2a57-4ef7-8357-3bd5016d79fe)

不过确实体验上我感觉还可以提升，这块后面可以好好探讨一下。

其实就是我说的：`提升 ConnectModal 的体验，连接钱包是最关键的用户体验链路。要更好更多的支持各种钱包的连接。除了当前的 simple 和 normal 的模式外提供更好体验的交互。`

**Comment by yutingzhao1991** [comment]
A idea about a new component CryptoPay https://github.com/ant-design/ant-design-web3/issues/415

**Comment by yutingzhao1991** [comment]
hot chain:
- solana
- near
- sui
- cosmos
